### ISSUE TITLE
Qat the Cat - an educational game

### ISSUE THREAD

**Issue by NandiBee** [qiskit-community/qiskit-camp-africa-19#32]
# Abstract
Video games are an excellent way to engage and interact with players. The aim of this project is to create a demo, prototype or vertical slice of a game that uses IBM Qiskit to demonstrate quantum computing concepts, create exposure and awareness to further interest in the field.

# Description
Qat the Cat is an educational puzzle game that seeks to invite players to experience using Qiskit and gain exposure to quantum computing and basic quantum mechanics. The goal is to make more people realise that quantum computers are accessible now due to IBM's Qiskit API.

# Members
 - @NandiBee
 - @Quanticnova
 - @dimpho0
 - @Daniela-gumme
 - @annvuyanzi
 - Qiskit Coach: @VicPinaCanelles

# Deliverable
Application prototype - Web or mobile or desktop

# GitHub repo
https://github.com/NandiBee/QCamp

**Comment by omarcostahamido** [comment]
_(...)
enter your full name in the field Name of your [profile](https://github.com/settings/profile).
(...)
Open an issue in this repo (using the [new project](https://github.com/qiskit-community/qiskit-camp-africa-19/issues/new?assignees=&labels=members+wanted&template=new-project-template.md&title=Project+name) template) and explain your idea in some paragraphs.
(...)_

**Comment by SsinghZAR** [comment]
I'd like to work on this project

**Comment by rodneyosodo** [comment]
@NandiBee  maybe something like this?
👇

**Comment by rodneyosodo** [comment]
# Abstract
Video games are an excellent way to engage and interact with players. The aim of this project is to create a demo, prototype or vertical slice of a game that uses IBM Qiskit to demonstrate quantum computing concepts, create exposure and awareness to further interest in the field.

# Description 
To create a demo of a game that uses Qiskit to demonstrate quantum computing concepts, create exposure and awareness to further interest in the field.

# Members


 - @NandiBee  
 - @githubhandle - Slack: `@slackhandle` email: `example@example.com`
 - Qiskit Coach: @omarcostahamido  

# Deliverable
Application - Web or mobile or desktop

# GitHub repo

**Comment by ghost** [comment]
I would like to work on this project.

**Comment by omarcostahamido** [comment]
This is looking good. I would love to be a coach, but I must say that @VicPinaCanelles is a great coach to have for this project.

**Comment by SsinghZAR** [comment]
@qcamp

**Comment by quantumjim** [comment]
If you want someone far away to bounce ideas off of, let me know.

**Comment by 1ucian0** [comment]
Dimpho Majatladi 
Ann vuyanzi 
Adano daniela  
Nandi Bau 
Siddarth Singh 

coach: vicente

**Comment by ghost** [comment]
@qcamp

**Comment by Daniela-gumme** [comment]
@qcamp

**Comment by annvuyanzi** [comment]
@qcamp

**Comment by QuantumVic** [comment]
Congrats on the honorable mention guys ! Since we got it, we should make playing as accessible as possible ASAP in case anyone from the Camp wants to play it, right? I'd like to write a nice README if you agree, since I still have to spend 3 more hours in the airport :)

**Comment by omarcostahamido** [comment]
me -> standing in line waiting to play it :|

**Comment by Daniela-gumme** [comment]
I agree.

Regards

**Comment by annvuyanzi** [comment]
Thank you. Yes i agree. We need to continue with the project. I got
intrest.

**Comment by QuantumVic** [comment]
Please add me as a collaborator in the project repository! @NandiBee

**Comment by NandiBee** [comment]
Yes! Just sent an invite, sorry for being late   From: Vicente Pina CanellesSent: Sunday, 15 December 2019 13:29To: qiskit-community/qiskit-camp-africa-19Cc: Nandi Bau; MentionSubject: Re: [qiskit-community/qiskit-camp-africa-19] Qat the Cat - an educational game (#32) Please add me as a collaborator in the project repository! @NandiBee—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
